docs/content/documentation/content/shortcodes.md

```diff
@@ -228,18 +231,18 @@ Mostly useful when combined with `load_data` or `resize_image`.
 
 ## Examples
 
-Here are some shortcodes for inspiration.
+Here are some sample shortcodes for inspiration.
 
 ### YouTube
 
 Embed a responsive player for a YouTube video.
 
 The arguments are:
 
-- `id`: the video id (mandatory)
-- `playlist`: the playlist id (optional)
-- `class`: a class to add to the `<div>` surrounding the iframe
-- `autoplay`: when set to "true", the video autoplays on load
+* `id`: the video id (mandatory)
+* `playlist`: the playlist id (optional)
+* `class`: a class to add to the `<div>` surrounding the iframe
+* `autoplay`: when set to "true", the video autoplays on load
```

Comment:
Can we revert the `-` -> `*` changes? It's the same meaning and it causes unnecessary diff